### PR TITLE
Renamed Dataprep Endpoints videos --> files

### PR DESCRIPTION
**Pull request by okhleif-10**
## Description

This PR goes along with [this PR in Comps](https://github.com/mhbuehler/GenAIComps/pull/3). The purpose of this pull request is to rename the dataprep endpoint variables to match the renaming done in the backend.

## Issues

N/A

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

Describe the tests that you ran to verify your changes.

Running the curl commands and checking logs has verified both endpoints are being called successfully under the new names.

### CODE REVIEW

**Review by mhbuehler — APPROVED** [review]
LGTM

File: MultimodalQnA/docker_compose/intel/cpu/xeon/README.md [hunk 1]
```diff
@@ -305,7 +306,7 @@ Also, you are able to get the list of all videos that you uploaded:
 ```bash
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:306]: Could you also update the text around this curl command. For example the line above this says "Also, you are able to get the list of all videos that you uploaded:" (change `videos` ➡️ `files`)

And then below the `curl` command it's also specifically saying `video`.

File: MultimodalQnA/docker_compose/intel/cpu/xeon/README.md [hunk 2]
```diff
@@ -322,7 +323,7 @@ To delete all uploaded videos along with data indexed with `$INDEX_NAME` in REDI
...
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:323]: Same with this, the markdown is talking about videos specifically.

File: MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md [hunk 3]
```diff
@@ -256,7 +257,7 @@ Also, you are able to get the list of all videos that you uploaded:
 ```bash
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md:257]: Same here

File: MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md [hunk 4]
```diff
@@ -273,7 +274,7 @@ To delete all uploaded videos along with data indexed with `$INDEX_NAME` in REDI
...
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md:274]: And here

**Review by dmsuehir — CHANGES_REQUESTED** [review]

**Review by dmsuehir — CHANGES_REQUESTED** [review]
Can you also update:
* MultimodalQnA/tests/test_compose_on_xeon.sh
* MultimodalQnA/tests/test_compose_on_gaudi.sh

These have references to `get_videos` and `delete_videos`

**Review by dmsuehir — APPROVED** [review]
LGTM, thanks

**Pull request merged**